app/src/test/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheetControllerTest.kt

```diff
@@ -114,4 +114,20 @@ class RemoveAppointmentSheetControllerTest {
         MovedToPrivatePractitioner,
         Other)
   }
+
+  @Test
+  fun `when done is clicked, and a "Patient has already visited" reason is selected, then mark as already visited should be called`() {
```

Comment:
Rename test to `when done is clicked, and a "Patient has already visited" reason is selected, then appointment should be marked as visited`?

Reply:
Done